### PR TITLE
feat: subseries

### PR DESCRIPTION
**Pull request by rjsparks**

### CODE REVIEW

File: ietf/templates/doc/index_subseries.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{% block title %}{{type.name}}s{% endblock %}
+{% block content %}
+  {% origin %}
+  <h1>{{type.name}}s</h1>
+  {% for doc in docs %}
+    <div class="card mb-3" id={{doc.name}}>
+    <div class="card-header"><a href="{% url 'ietf.doc.views_doc.document_main' name=doc.name %}">{{doc.name}}</a></div>
+      <div class="card-body">
+        {% for rfc in doc.contains %}
+          <p><a href="{% url 'ietf.doc.views_doc.document_main' name=rfc.name %}"">{{rfc.name}}</a> : {{rfc.title}}</p>
```
Comment (jennifer-richards) [inline ietf/templates/doc/index_subseries.html:14]: Extra `"` in the `<a...>` opening tag

**Review by jennifer-richards — CHANGES_REQUESTED** [review]
A couple questions and an HTML lint inline. Also the tests look like they need to be updated - at least, I don't think those are random failures.

In the search view and in the individual subseries pages, contained rfcs are listed in arbitrary order - some are increasing rfc numbers, some are decreasing, and I imagine there are probably some mixed. If that's not significant (and I don't think it is based on how the contents are modeled), it'd be nice to sort those.

File: ietf/sync/rfceditor.py [hunk 2]
```diff
@@ -650,15 +653,47 @@ def parse_relation_list(l):
...
+
+            for a in also:
+                subseries_doc_name = a
+                subseries_slug=a[:3]
+                # Leaving most things to the default intentionally
+                # Of note, title and stream are left to the defaults of "" and none.
+                subseries_doc, created = Document.objects.get_or_create(type_id=subseries_slug, name=subseries_doc_name)
+                if created:
+                    if first_sync_creating_subseries:
+                        subseries_doc.docevent_set.create(type=f"{subseries_slug}_history_marker", by=system, desc=f"No history of this {subseries_slug.upper()} document is currently available in the datatracker before this point")
```
[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/sync/rfceditor.py:682]: intended both to be the same - thought mutated during work - will normalize.

[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/sync/rfceditor.py:682]: >(comment applies both here and below)

I don't see where this applies below?

[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/sync/rfceditor.py:682]: ah nevermind

**Review by jennifer-richards — COMMENTED** [review]
Changes you've made look good so far.

**Review by jennifer-richards — APPROVED** [review]

**Pull request merged**